src/main/java/com/iwa/Calculator/Config/DivideConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.iwa.Calculator.Config;
+
+import com.iwa.Calculator.Operation.Divide;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
```

Comment:
In this case is not neccessary to have to configuration files, you can put multiple beans per file, so you can merge this and the CalcualtorConfig one.